thirdparty/build-thirdparty.sh

```diff
@@ -1361,6 +1361,8 @@ build_icu() {
     unset CXXFLAGS
     unset CFLAGS
 
+    export LD_LIBRARY_PATH=${STARROCKS_GCC_HOME}/lib:${STARROCKS_GCC_HOME}/lib64:${LD_LIBRARY_PATH}
```

Comment:
worried about the LD_LIBRARY_PATH change affecting the entire build progress, can this be limited to just build_icu?

Reply:
an alternative way is to try to build the binary with -static-libgcc and -static-libstdc++, so the binary itself may not necessary to depend on any external libraries.